dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/StringConcatenateAssignmentNode.java

```diff
@@ -15,7 +15,10 @@
  * <pre>
  *   <em>variable</em> += <em>expression</em>
  * </pre>
+ *
+ * @deprecated Use {@code #AssignmentNode()} instead.
```

Comment:
Nobody has to use `AssignmentNode` instead and this seems like a reference to a constructor, which doesn't exist in this class. Say that `StringConcatenateAssignmentNode` isn't generated any more in CFGs and that instead an assignment and concatenation is generated.

Reply:
Okay, thanks!